tex/artikel/Gleichstellungsbuero.tex

```diff
@@ -1,35 +1,35 @@
+\begin{wrapfigure}[6]{r}{3.4cm}
+    \vspace*{-0.3cm}
+    \includegraphics[width=\linewidth]{res/gst_buero.jpg}
+\end{wrapfigure}
 
-Das Physikerinnen-Café findet immer statt, wenn im Allgemeinen Physikalischen Kolloquium eine Rednerin spricht. Am selben Tag ihres Vortrags treffen wir uns vorab und sprechen sowohl über ihren Karriereweg als auch über geschlechterbezogene Probleme, denen sie während ihres eigenen Werdegangs begegnet ist. Meistens haben diese schon weiter fortgeschrittenen Wissenschaftlerinnen super Tipps für uns Studentinnen. \\
+Die ersten Veranstaltungen der O-Woche sind schon vorbei, alles ist neu. Der Hörsaal schaut ganz anders aus als ein Klassenzimmer und sicherlich habt ihr auch schon eure ganzen Mitstudierenden gesehen, die, genau wie ihr, gerade einen neuen Lebensabschnitt mit dem Studium der Physik beginnen. Doch habt ihr mal darauf geachtet, wie die Verteilung der Geschlechter ausschaut? Wenn nicht, probiert es mal! Ihr werdet feststellen, dass vor allem Männer stark vertreten sind. Die Anzahl der weiblichen Physikerinnen nimmt entsprechend des Kaskadenprinzips mit steigenden Karrierestufen sogar immer weiter ab: Es gibt noch weniger Doktorandinnen, weibliche Post-Docs und noch weniger Professorinnen. Warum ist das so? Welches strukturelle Problem liegt dem zugrunde? Und was können wir dafür tun, um den Frauenanteil in der Physik zu erhöhen? An wen wende ich mich bei sexueller Belästigung im Studium? Für alle diese Fragen sind wir vom Gleichstellungsbüro zuständig.
 
-Auch für die Themen sexualisierte Diskriminierung, Belästigung und Gewalt sind wir zuständig. Dabei haben wir immer ein offenes Ohr für euch, gehen mit euren Sorgen sowie Nöten diskret um und kennen die nötigen Ansprechpersonen, wenn weitere Schritte eingeleitet werden sollen. \\
+Zum Beispiel veranstalten wir regelmäßig Gender Breakfast Talks und Physikerinnen-Cafés. Dort können wir uns austauschen und unterstützen sowie über interessante und aktuelle Themen quatschen. Der Gender Breakfast Talk ist eine Veranstaltung explizit für alle Geschlechter – Gleichstellung ist nämlich kein Thema für Frauen allein. Während des Physikerinnen Cafés möchten wir jedoch eine für Frauen geschützte Umgebung schaffen und Hürden und Probleme besprechen, die speziell Frauen im Studium und in der Karriere begegnen.
 
-Alle Infos dafür findet ihr auf unserer Website \footnote{\url{www.uni-muenster.de/Physik/department/equality/}} oder bekommt ihr regelmäßig, indem ihr euch auf unseren entspannten Newsletter setzt. Dies geht ganz einfach über diesen QR-Code: \\
```

Comment:
[...] Website\footnote{\url{...}} [...] 

'footnote' also direkt hinter das Wort (wurde mir so eingeprügelt)

Reply:
Jo, ist bereits in der geänderten Version dieses Absatzes so (die grüne Zeile 23). Ja, das mit den roten und grünen Zeilen ist diesmal tatsächlich etwas unübersichtlich, weil leider die roten Zeilen (also die Zeilen vor der Änderung, mit Minus am Anfang) nicht unbedingt in der Nähe der grünen Zeilen (also den Zeilen nach der Änderung, mit plus am Anfang) sind